TempleApp/src/main/java/example/TempleApp/Activities/ReadSingleData.java

```diff
@@ -63,12 +64,12 @@ public void onCheckedChanged(RadioGroup group, int checkedId) {
                     case R.id.radio_donate:
                         totalLayout.setVisibility(View.VISIBLE);
                         flag = 0;
-                        Toast.makeText(getBaseContext(), getString(R.string.selected_donate), Toast.LENGTH_LONG).show();
+                        Toast.makeText(getBaseContext(), "Selected To Donate Money", Toast.LENGTH_LONG).show();
```

Comment:
Any particular reason for replacing the string resource with hardcoded string? If not please revert back to using string resources.